cmd/azurerm.go

```diff
@@ -68,12 +68,12 @@ var (
 
 			if hclOut != nil {
 				logger.Log("msg", "initializing HCL writer")
-				hclW = hcl.NewWriter(hclOut)
+				hclW = hcl.NewWriter(hclOut, &writer.Options{Interpolate: !viper.GetBool("no-interpolate")})
 			}
 
 			if stateOut != nil {
 				logger.Log("msg", "initializing TFState writer")
-				stateW = state.NewWriter(stateOut)
+				stateW = state.NewWriter(stateOut, &writer.Options{Interpolate: !viper.GetBool("interpolate")})
```

Comment:
Is there a flag `interpolate`?

Reply:
No, I messed up with my first implementation !